financial-services-accelerator/components/org.wso2.financial.services.accelerator.common/src/main/java/org/wso2/financial/services/accelerator/common/constant/FinancialServicesConstants.java

```diff
@@ -269,5 +266,24 @@ public class FinancialServicesConstants {
     public static final String DOT_SEPARATOR = ".";
     public static final String PRODUCTION = "PRODUCTION";
     public static final String SANDBOX = "SANDBOX";
+
+// NEw
```

Comment:
typo

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/108/commits/74e401a23f66920af64aee5717dd4ccc8a075948